utils/mkhtml.py

```diff
@@ -188,71 +179,169 @@ def download_git_commit(url, response_format, *args, **kwargs):
         )
 
 
-def get_last_git_commit(src_dir, is_addon, addon_path):
-    """Get last module/addon git commit
+def parse_git_commit(
+    commit,
+    git_log={"commit": "unknown", "date": "unknown"},
+):
+    """Parse Git commit
 
-    :param str src_dir: module/addon source dir
-    :param bool is_addon: True if it is addon
-    :param str addon_path: addon path
+    :param str commit: commit message
+    :param dict git_log: dict which store last commit and commnit
+                         date
 
-    :return dict git_log: dict with key commit and date, if not
-                          possible download commit from GitHub API server
-                          values of keys have "unknown" string
+    :return dict git_log: dict which store last commit and commnit date
+    """
+    git_log["commit"], commit_date = commit.strip().split(",")
+    git_log["date"] = format_git_commit_date_from_local_git(
+        commit_datetime=commit_date,
+    )
+    return git_log
+
+
+def get_git_commit_from_file(
+    src_dir,
+    git_log={"commit": "unknown", "date": "unknown"},
```

Comment:
Here perhaps okay, but it is generally bad practice to use mutable values as defaults.

https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
Sure, you're right.